tests/tests.js

```diff
@@ -450,7 +450,7 @@ exports.defineAutoTests = function () {
                         expect(transfer.onprogress).toHaveBeenCalled();
 
                         // check that there is no file
-                        root.getFile(localFilePath, null, unexpectedCallbacks.fileSystemWin, done);
+                        root.getFile(fileName, null, unexpectedCallbacks.fileSystemWin, done);
```

Comment:
Despite the fact that [the docs](https://github.com/apache/cordova-plugin-file/blob/ab42f3e7c6c2282ac303e32260e8d68fdd2e18e3/www/DirectoryEntry.js#L98) says it takes a relative or an absolute path (by the way I can't find any File plugin test for an absolute path), the use of `localFilePath` (which is `file:///` on Android and `ms-appdata:///` on Windows) causes a fail with `FileError.ENCODING_ERR = 5` instead of `FileError.NOT_FOUND_ERR = 1`.
Probably the absolute path should be absolute to the fs root like `/dir/file.txt` and not a native path.